tests/pytest/ztp-left-shifting/sno-hub/check-hub-installation.py

```diff
@@ -0,0 +1,45 @@
+import os
+import time
+import requests
+import pytest
+
+kubeconfig=os.getenv("KUBECONFIG")
+ocp_hub_version=os.getenv("OCP_HUB_VERSION")
+mch_ns=os.getenv("MCH_NAMESPACE")
+
+@pytest.mark.parametrize("endpoint", [
+    {"cmd": f"oc --kubeconfig {kubeconfig} whoami --show-console", "response": 200},
+    {"cmd": f"oc --kubeconfig {kubeconfig} get managedcluster local-cluster -ojsonpath='{{.spec.managedClusterClientConfigs[0].url}}'", "response": 403},
+])
+def test_http_endpoint(bash, endpoint):
```

Comment:
It's not so clear why you need to use parameterize here.

Reply:
As I said before, I just moved the code that was previously in the Openshift-CI step here for a conceptual test. At the moment I'm not that interested in this test being really very comprehensive, just that it works.

When we have the Openshift-CI part integrated, we can come back to this and discuss what tests to run to ensure that the Hub cluster and its components have been correctly installed.

My idea is that, later on, we can adjust the tests to run from eco-ci-cd.

Reply:
Sounds good